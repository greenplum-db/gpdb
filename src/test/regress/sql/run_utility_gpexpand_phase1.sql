-- start_matchsubs
-- m/^\S+ (.*):.*:.*-(\[\S+\]):-/
-- s/^\S+ (.*):.*:.*-(\[\S+\]):-/TIMESTAMP $1:HOST:USER-$2:-/
-- end_matchsubs

\! echo 'EXPANSION_PREPARE_STARTED:<path> to inputfile' > $MASTER_DATA_DIRECTORY/gpexpand.status
\! gpcheckcat
\! gpconfig -r gp_debug_linger
\! gppkg --query no-such-package
\! rm $MASTER_DATA_DIRECTORY/gpexpand.status
