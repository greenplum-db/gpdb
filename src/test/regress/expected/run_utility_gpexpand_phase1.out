-- start_matchignore
-- m/.*-/
-- end_matchignore
\! echo 'EXPANSION_PREPARE_STARTED:<path> to inputfile' > $MASTER_DATA_DIRECTORY/gpexpand.status
\! gpcheckcat
20190107:14:45:14:022869 gpcheckcat:zlv:gpadmin-[ERROR]:-ERROR: Usage of gpcheckcat is not supported while thecluster is in a reconfiguration state, exit gpcheckcat
\! gpconfig -r gp_debug_linger
20190107:14:45:14:022890 gpconfig:zlv:gpadmin-[ERROR]:-ERROR: Usage of gpconfig is not supported while thecluster is in a reconfiguration state, exit gpconfig
\! gppkg
20190107:14:45:14:022909 gppkg:default-[ERROR]:-ERROR: Usage of gppkg is not supported while thecluster is in a reconfiguration state, exit gppkg
\! rm $MASTER_DATA_DIRECTORY/gpexpand.status
